src/components/Nav/Dropdown.js

```diff
@@ -140,54 +144,53 @@ const NavDropdown = ({ section, hasSubNav }) => {
   const ariaLabel = section.ariaLabel || section.text
 
   return (
-    <NavListItem ref={ref} aria-label={translateMessageId(ariaLabel, intl)}>
-      <DropdownTitle
-        dir={isPageRightToLeft ? "auto" : "ltr"}
-        onClick={() => setIsOpen(!isOpen)}
-        onKeyDown={onKeyDownHandler}
-        tabIndex="0"
-      >
-        <Translation id={section.text} />
-        {section.component ? (
-          <Online />
-        ) : (
+    <DropdownContext.Provider
+      value={{ isOpen, toggle, close, tabInteractionHandler }}
+    >
+      <NavListItem ref={ref} aria-label={translateMessageId(ariaLabel, intl)}>
+        <DropdownTitle
+          dir={isPageRightToLeft ? "auto" : "ltr"}
+          onClick={() => toggle()}
+          onKeyDown={onKeyDownHandler}
+          tabIndex="0"
```

Comment:
We don't currently signify that this is an interactive element.

```suggestion
          tabIndex="0"
          role="button"
          aria-expanded={isOpen ? "true" : "false"}
```

Reply:
Nice! done